tools/devops/device-tests/templates/publish-html.yml

```diff
@@ -27,8 +27,10 @@ steps:
 # Use the cmdlet to post a new summary comment. The cmdlet checks if we have the TestSummary.md file or not. It will also add the appropriate links to the commnet. 
```

Comment:
```suggestion
# Use the cmdlet to post a new summary comment. The cmdlet checks if we have the TestSummary.md file or not. It will also add the appropriate links to the comment. 
```